config/vendor-api-v1.yml

```diff
@@ -831,6 +832,11 @@ components:
           description: The subject studied
           maxLength: 256
           example: History and Politics
+        subject_code:
+          type: string
+          description: The HECoS code for the subject studied, if available
```

Comment:
Is the specificity of saying HECoS useful here? Are they going to be extensible to all types of qualification, or just limited to GCSEs and A levels?

Reply:
Might be better to be explicit and add GCSE and A levels, but something to check with James I think to be sure